tests/conformance_ruby/standard_filter_test.rs

```diff
@@ -401,11 +412,17 @@ fn test_uniq_empty_array() {
 }
 
 #[test]
-#[should_panic] // liquid-rust#335
 fn test_compact_empty_array() {
     assert_eq!(v!([]), filters!(Compact, v!([]), v!("a")));
 }
 
+#[test]
+fn test_compact_invalid_property() {
```

Comment:
Same as above: I'd prefer new tests not to be put in here

Reply:
I got this test from [here](https://github.com/Shopify/liquid/blob/14cd011cb5eef042af192ad5294c8f0e1ae89f06/test/integration/standard_filter_test.rb#L345), so I think this is a correct place.

Reply:
Looks like we need to do an audit at some point for updating.  I forked the tests about a year ago and they added this test 7 months ago.